Contrib/Fastcluster/testdata/sdf2smi.py

```diff
@@ -0,0 +1,10 @@
+import argparse
```

Comment:
You could do the work that this file does a tiny bit more simply using an RDKit `SmilesMolWriter`, but doing it with Python is fine too.

Reply:
I tried to change sdf2smi.py by using SmilesMolWriter. Thank you for your comment.